Kerdy/Kerdy/MyProfile/VC/ActivityEditHalfVC.swift

```diff
@@ -0,0 +1,292 @@
+        btn.setTitleColor(.black, for: .normal)
+        btn.layer.borderWidth = 2.0
+        btn.layer.borderColor = UIColor(named: "kerdy_gray01")?.cgColor
+        btn.layer.cornerRadius = 10.0
+        btn.contentHorizontalAlignment = .left
+        btn.addTarget(self, action: #selector(educationBtnTapped), for: .touchUpInside)
+        return btn
+    }()
+    
+    private lazy var grabBar: UIImageView = {
+        let bar = UIImageView(image: .icGrabbar)
+        return bar
+    }()
+    
+    private lazy var nextBtn: UIButton = {
+        let button = UIButton()
+        button.titleLabel?.font = .nanumSquare(to: .regular, size: 16)
+        button.backgroundColor = .kerdyMain
+        button.layer.cornerRadius = 15
+        button.setTitle("다음", for: .normal)
+        button.addTarget(self, action: #selector(nextBtnTapped), for: .touchUpInside)
+        return button
+    }()
+    
+    private lazy var containerView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        view.layer.cornerRadius = 16
+        view.clipsToBounds = true
+        return view
+    }()
+    
+    private lazy var dimmedView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .black
+        view.alpha = maxDimmedAlpha
+        return view
+    }()
+    
+    private let maxDimmedAlpha: CGFloat = 0.6
+    private let defaultHeight: CGFloat = 300
+    private let dismissibleHeight: CGFloat = 200
+    private let maximumContainerHeight: CGFloat = UIScreen.main.bounds.height - 64
+    private var currentContainerHeight: CGFloat = 300
+    
+    private var containerViewHeightConstraint: NSLayoutConstraint?
+    private var containerViewBottomConstraint: NSLayoutConstraint?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUI()
+        setLayout()
+        setTabGesture()
+        setupPanGesture()
+    }
+    
+    @objc func handleCloseAction() {
+        animateDismissView()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        animateShowDimmedView()
+        animatePresentContainer()
+    }
+    
+    private func setUI() {
+        view.backgroundColor = .clear
+    }
+    
+    private func setTabGesture() {
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(self.handleCloseAction))
+        dimmedView.addGestureRecognizer(tapGesture)
+    }
+    
+    private func setLayout() {
+        view.addSubview(dimmedView)
+        view.addSubview(containerView)
+        
+        dimmedView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        containerView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+        }
+        containerView.addSubview(baseView)
+    
+        baseView.addSubviews(clubBtn, educationBtn, titleLabel, notesLabel, grabBar, nextBtn)
```

Comment:
```suggestion
        baseView.addSubviews(
            clubBtn,
            educationBtn,
            titleLabel, 
           notesLabel,
           grabBar,
           nextBtn
           )
```
와 같이 작성해도 좋을 것 같습니다!